### PR TITLE
MCLabels in clusters; Fix to read full raw payload; remove Publisher

### PR DESCRIPTION
**Pull request by peressounko**

### CODE REVIEW

File: DataFormats/Detectors/PHOS/include/DataFormatsPHOS/Cluster.h [hunk 1]
```diff
@@ -82,7 +78,7 @@ class Cluster
   float mTime = 0.;              ///< Time of the digit with maximal energy deposition
   float mDistToBadChannel = 999; ///< Distance to nearest bad crystal
 
-  ClassDefNV(Cluster, 1);
+  ClassDefNV(Cluster, 2);
```
Comment (shahor02) [inline DataFormats/Detectors/PHOS/include/DataFormatsPHOS/Cluster.h:81]: It does not harm, but the class version need to be changed if the class layout was modified. Modifying some methods does not require this.

**Review by shahor02 — APPROVED** [review]
@peressounko : looks fine (but I did not run it), is this ready to be merged?

File: DataFormats/Detectors/PHOS/include/DataFormatsPHOS/Cluster.h (hunk 1 above)
Comment (peressounko) [reply DataFormats/Detectors/PHOS/include/DataFormatsPHOS/Cluster.h:81]: I removed datamember: ref to MCLabel. Normally one should increase version in this case. But as we not not have a lot of data, we can keep 1.

**Comment by peressounko** [comment]
Dear @shahor02, 
no, test showed problems in raw reading/writing(?) Should be fixed before merging.

File: DataFormats/Detectors/PHOS/include/DataFormatsPHOS/Cluster.h (hunk 1 above)
Comment (shahor02) [reply DataFormats/Detectors/PHOS/include/DataFormatsPHOS/Cluster.h:81]: Sorry, missed this, the increment is indeed needed (and does not harm anyway).

**Review by shahor02 — CHANGES_REQUESTED** [review]
OK, flagging by Request changes to avoid merging by others.

**Comment by peressounko** [comment]
Dear @shahor02, now code ready for merging. Could you please approve?

**Review by shahor02 — APPROVED** [review]

**Pull request merged**